ui/StatusQ/include/StatusQ/keychain.h

```diff
@@ -14,6 +14,7 @@ class Keychain : public QObject {
 
     Q_PROPERTY(QString service READ service WRITE setService NOTIFY serviceChanged)
     Q_PROPERTY(bool loading READ loading NOTIFY loadingChanged)
+    Q_PROPERTY(bool available READ available CONSTANT)
```

Comment:
Is it really constant? What if it's available at the start but then later on, the user disables the feature globally via the System Settings?

Reply:
Sure, something that can definitely be done later (I assume the first option should be available)

Reply:
I did a quick research, there seem to be no signalling.
Looks like the proper solution is to reevaluate it when window becomes active.

How simple would it be now to implement in C++ 🙂